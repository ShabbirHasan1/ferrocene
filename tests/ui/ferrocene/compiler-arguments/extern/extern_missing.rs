// check-fail
// aux-build:some_crate.rs
// compile-flags:--extern some_other_crate
// edition:2018

fn main() {
    ::some_crate::hello(); //~ ERROR
}

// ferrocene-annotations: um_rustc_extern
