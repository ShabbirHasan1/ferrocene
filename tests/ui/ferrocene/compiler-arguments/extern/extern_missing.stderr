error[E0433]: failed to resolve: could not find `some_crate` in the list of imported crates
  --> $DIR/extern_missing.rs:7:7
   |
LL |     ::some_crate::hello();
   |       ^^^^^^^^^^ could not find `some_crate` in the list of imported crates

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
